roles/ceph-ansible/tasks/main.yml

```diff
@@ -0,0 +1,138 @@
+---
+# ceph-ansible package is not in Epel yet. This playbook expects that
+# ceph-ansible is preinstalled or available in enabled yum repositories.
+# TODO do apt installation
+- name: Install ceph-ansible package
+  yum:
+    name=ceph-ansible
```

Comment:
I see that you just install `ceph-ansible` package without enabling particular repository for it. That is good.

That said, I would like to know which ceph-ansible package you tested this role with?